Week3/MAKEME.md

```diff
@@ -54,59 +73,126 @@ let favoriteAnimals = ['blowfish', 'capricorn', 'giraffe'];
+5. Can you write the following without the `if` statement, but with just as a single line with `console.log(...);`?
 
-6. Beautify your html page with css, add sources and alts to each of the images.
- 
-7. Download book covers for each book, construct a new Object which has as keys the bookId's again, and as value the path to the image source (e.g. `{"harry_potter_blabla": "./img/harry_potter_blabla.jpg", ...}`). Now loop over these entries (_hint: `Object.keys(objectName)` gives you an array containing the keys_). Then write a function which places an image at the corresponding `li` element. Remember that Objects are not ordered, so you cannot guarantee that the first key is the first `li` element. (_Hint: you could give each `li` item an `id` tag by modifying the function you made before_)
+    ```js
+    if (3 === 3) {
+        console.log("true")
+    } else {
+        console.log("false")
+    }
+    ```
 
-```
-How to hand in your homework:
-• Upload your homework in your "hyf-javascript1" Github repository. Make sure to create a new folder "week3" first. 
-• Upload your homework files inside the week3 folder and write a description for this “commit”.
-• Your hyf-javascript1/week3 should now contain an index.html, main.css and a script.js file (and the images folder)
-• Place the link to your repository folder in Trello.
-```
+6. Create a function called `vehicle`, like before, but takes another parameter called age, so that `vehicle("blue", 1, 5)` prints "a blue used car"
 
-## Step 5: **FreeCodeCamp challenges:**
+7. Make a list of vehicles, you can add `"motorbike"`, `"caravan"`, `"bike"`, or more.
 
-_Deadline Saturday_
+8. How do you get the third element from that list?
+
+9. Change the function `vehicle` to use the list of question 7. So that `vehicle("green", 3, 1)` prints "a green new caravan".
+
+10. Use the list of vehicles to write an advertisement. So that it prints something like: `"Amazing Joe's Garage, we service cars, motorbikes, caravans and bikes."`. (Hint: use a `for` loop.)
+
+11. What if you add one more vehicle to the list, can you have that added to the advertisement without changing the code for question 10?
+
+12. Create an empty object.
+
+13. Create an object that contains the teachers that you have had so far for the different modules.
 
-- https://www.freecodecamp.com/challenges/declare-javascript-objects-as-variables
-- https://www.freecodecamp.com/challenges/make-instances-of-objects-with-a-constructor-function
-- https://www.freecodecamp.com/challenges/make-unique-objects-by-passing-parameters-to-our-constructor
-- https://www.freecodecamp.com/challenges/make-object-properties-private
+14. Add a property to the object you just created that contains the languages that they have taught you.
 
+15. We saw that we can pass functions as arguments to other functions. Your task is to write a function that takes another function as an argument and runs it. 
```

Comment:
This assignment is about callbacks which we now have scheduled for week 6. Move it there?